driver-core/src/main/com/mongodb/client/model/vault/EncryptOptions.java

```diff
@@ -173,12 +175,12 @@ public String getQueryType() {
     /**
      * The RangeOptions
      *
-     * <p>It is an error to set RangeOptions when the algorithm is not "RangePreview".
+     * <p>It is an error to set RangeOptions when the algorithm is not "Range".
      * <p>Note: The Range algorithm is experimental only. It is not intended for public use. It is subject to breaking changes.
```

Comment:
Is this still considered "experimental"?

Reply:
No, it should be marked as 'unstable.' I've removed the 'experimental' label and updated the status accordingly.